### PR TITLE
Disable mob spawn on the chisel blocks

### PR DESCRIPTION
**Pull request by Zereff06**
All gt blocks have anti-spawn protection, why don't the decorative blocks? I think it would be okay if mobs were not spawned on the base, but only outside of it. Forever watching the lights on the base or the mechanisms forbidding spawn is tiresome. I think it would be great if the base will not be mobs at all, because where do they come from?

### CODE REVIEW

**Comment by Dream-Master** [comment]
?

**Comment by Zereff06** [comment]
@Dream-Master now it's done.

**Comment by tyrael93** [comment]
Yeah no
I think this is big enough to warrant more than just a "nope, it's done"

File: src/main/java/team/chisel/block/BlockCarvable.java [hunk 1]
```diff
@@ -179,4 +180,8 @@ public String func_150496_b() {
 				return this.soundNamePlace;
 		}
 	}
+    @Override
+    public boolean canCreatureSpawn(EnumCreatureType type, IBlockAccess world, int x, int y, int z) {
+        return false;
```
Comment (mitchej123) [inline src/main/java/team/chisel/block/BlockCarvable.java:185]: This should be a config.

**Review by mitchej123 — CHANGES_REQUESTED** [review]
Needs to be a (cached) config option.  I don't think globally disabling it without an option is the way to go.

**Review by OneEyeMaker — APPROVED** [review]

**Review by mitchej123 — APPROVED** [review]

**Pull request merged**